README.md

```diff
@@ -36,19 +36,65 @@ scrape_configs:
     metrics_path: /export
     params:
       command: [check_load] # Run the check_load command.
+    relabel_configs:
+      - source_labels: [__address__]
+        target_label: __param_target
+      - source_labels: [__param_target]
+        target_label: instance
+      - target_label: __address__
+        replacement: 127.0.0.1:9275 # Nrpe exporter.
     static_configs:
       - targets: # Targets to run the specified command against.
         - '127.0.0.1:5666'
         - 'example.com:5666'
+
+  - job_name: nrpe_check_apt
+    params:
+      command: [check_apt]
```

Comment:
Is this a command that all users are likely to have out of the box? Examples shouldn't require specific setup by the user (e.g. running a Debian-based OS) .

In general, this example doesn't seem to be adding anything.

Reply:
Good point.  Using something like check_mysql_query would have been better as it is part of the standard  plugin distribution.  However, as we removed the long polling, this example, even with a generic command, adds little value.  I changed the example to use check_load.     It I get inspired and can make the time, I'll learn go and extend the exporter from a wire-protocol converter to something that adapts nagios time semantics.